### PR TITLE
Fix broken link, fix & add redirects

### LINKED ISSUE #10121: Broken links to kubelet-tls-bootstrapping
### This is a

- [ ] Feature Request
- [x] Bug Report

### Problem

There's a [broken link](https://kubernetes.io/docs/tasks/tls/certificate-rotation/#overview) for kubelet certificate rotation. 

It looks like the file URL was changed during the [1.6 release](https://github.com/kubernetes/website/pull/3617) and handled with a [redirect](https://github.com/kubernetes/website/blob/55eee978cdfbfb3e27752d138b6bc5d925c7d658/static/_redirects#L266).

The redirect was subsequently broken by changes introduced in #9332. 

### Proposed Solution

1. Update `/docs/tasks/tls/certificate-rotation/` with the correct link.

2. Fix other possible breaks by updating `/static/_redirects/` with the following changes:
```
/docs/tasks/administer-cluster/certificate-rotation/     /docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping/ 301
/docs/admin/kubelet-tls-bootstrapping/     /docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping/      301
```

### PR DESCRIPTION
**Pull request by zacharysarah**
Fixes #10121 

This PR fixes a broken link and updates `/static/_redirects/` to prevent other possible related breakages.

### CODE REVIEW

**Comment by zacharysarah** [comment]
### Test in staging

Click the link for kubelet certificate rotation:

- [ ] Current: https://kubernetes.io/docs/tasks/tls/certificate-rotation/#overview

- [ ] Preview: https://deploy-preview-10122--kubernetes-io-master-staging.netlify.com/docs/tasks/tls/certificate-rotation/#overview

File: content/en/docs/tasks/tls/certificate-rotation.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ default, these certificates are issued with one year expiration so that they do
 not need to be renewed too frequently.
 
 Kubernetes 1.8 contains [kubelet certificate
-rotation](/docs/tasks/administer-cluster/certificate-rotation/), a beta feature
+rotation](/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping/), a beta feature
```
[the remark this answers is not in the document]
Comment (tengqm) [reply content/en/docs/tasks/tls/certificate-rotation.md]: Agree this link can be removed.

**Pull request merged**